src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,57 @@
+@F-001
+Feature: F-001: Get Document Metadata by Document ID
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-001
+  Scenario: must successfully get document metadata by document ID
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    And   a successful call [by another privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   it is submitted to call the [Get Document Metadata by Document ID] operation of [CCD Case Document AM API]
+    Then  a positive response is received
+    And   the response [contains the metadata for the document uploaded above]
+    And   the response has all other details as expected
+
+
+  @S-002
+  Scenario: must get an error response for a non-existing document ID
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a non-existing document ID]
+    And   it is submitted to call the [Get Document Metadata by Document ID] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-003
+  Scenario: must get an error response for a malformed document ID
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed document ID]
+    And   it is submitted to call the [Get Document Metadata by Document ID] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-004
```

Comment:
S-004 won't not be possible in this APIs functional testing scope as it is dependent upon the following element after integration with AM-450:
1. The document need to uploaded first.
2. The document metadata must be linked with an existing ccd case reference.
3. CCD case must be having the same document Id (in documentFiled>url) which is uploaded in the step #1 above.

Overall it this expects that a document must be uploaded as per the updated upload process which is very difficult to simulate for testing this API, therefore this scenario can only be tested manually. we can evaluate the situation again after full upload(3 legs) implementation.